src/App.jsx

```diff
@@ -31,8 +31,8 @@ import { girth as gTheme } from "./themes/girth.js";
 import { v4 as uuidv4 } from "uuid";
 import "./style.scss";
 import { BondType } from "./lib/Bond";
-import SwapGuru from "./views/Swap/SwapGuru";
-import SwapNFT from "./views/Swap/SwapNFT";
+import SwapGuru from "./views/Swap/SwapGuru/SwapGuru";
+import SwapNFT from "./views/Swap/SwapNFT/SwapNFT";
```

Comment:
Suggestion: Having `index.ts` under each JSX component (that exports default) makes importing much cleaner, this is somewhat redundant

```
Swap
  Swap.tsx
  index.ts

inside index.ts
export { default } from "./SwapGuru"
```